Sources/ProjectSpec/Scheme.swift

```diff
@@ -498,6 +498,8 @@ extension Scheme.Test: JSONEncodable {
             "language": language,
             "region": region,
             "coverageTargets": coverageTargets.map { $0.reference },
+            "captureScreenshotsAutomatically": captureScreenshotsAutomatically,
+            "deleteScreenshotsWhenEachTestSucceeds": deleteScreenshotsWhenEachTestSucceeds,
```

Comment:
Thanks. Sorry if it wasn't clear before, but these values should only be written to the dictionary if they are not their default value (true), like below. This is so generated specs only contain the required information

Reply:
Ah, got it! This makes perfect sense. Will make the change right away.